src/components/pages/Registration/RegisterFormComponents/StudentForm.js

```diff
@@ -144,7 +158,15 @@ export default function StudentForm({ studentArr, setStudentArr, idx }) {
           src={avi4}
           alt=""
         />
-        <button id="add-btn">+</button>
+        <input
+          type="file"
+          ref={fileInput}
+          style={{ display: 'none' }}
+          onChange={fileSelectedHandler}
+        />
+        <button id="add-btn" onClick={() => fileInput.current.click()}>
+          +
+        </button>
```

Comment:
I love that they will be able to add their own image. Everything looks clean here!